lib/models/Film.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+
+const filmSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true
+  },
+  studio: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Studio',
+    required: true
+  },
+  released: {
+    type: Number,
+    required: true,
+    maxlength: 4
```

Comment:
maxlength only works on type:String; how would you fix this to work for type:Number?